postgres/toggle-backup-activation.sh

```diff
@@ -94,4 +94,32 @@ else
     echo "" >> /etc/cron.d/backup_postgres_crontab
     service cron restart
     echo "PostgreSQL automatic backup schedule: ${POSTGRES_BACKUP_SCHEDULE}"
-fi
+
+    if [[ ${USE_WAL_E} -eq "$TRUE") ]]; then
+        echo "Installing envdir and Wal-e for PostgreSQL backup on S3..."
+        apt-get install -y libevent-dev python-all-dev daemontools lzop pv curl --quiet=2 > /dev/null
+        python3 -m pip install wal-e aws --quiet pip
+
+        # Find EC2 region
+        EC2_AVAIL_ZONE=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone) #AWS zone
+        EC2_REGION=$(echo $EC2_AVAIL_ZONE | sed 's/[a-z]$//') #AWS region
+
+        # Add envdir for Wal-e backup
+        mkdir -p ${POSTGRES_DATA_DIR}/wal-e.d/env
```

Comment:
From where does `$POSTGRES_DATA_DIR` come from?

According to `env` in PostgreSQL, it is `PGDATA` 
```
root@postgres:/# env|grep PGDATA
PGDATA=/var/lib/postgresql/data

root@postgres:/# env|grep POSTGRES_DATA_DIR
PGDATA=/var/lib/postgresql/data
root@postgres:/#
root@postgres:/# env|grep POSTGRES_DATA_DIR
root@postgres:/#
```